pyproject.toml

```diff
@@ -98,3 +98,18 @@ all = [
 [tool.isort]
 profile = "black"
 known_third_party = ["fastapi", "pydantic", "starlette"]
+
+[tool.pytest.ini_options]
+addopts = [
+  "--strict-config",
+  "--strict-markers",
+]
+xfail_strict = true
+junit_family = "xunit2"
+filterwarnings = [
+    "error",
+    'ignore:"@coroutine" decorator is deprecated since Python 3\.8, use "async def" instead:DeprecationWarning',
```

Comment:
I see that without this ignore it fails on Python 3.8 because `aiofiles` uses `@coroutine`. I imagine this will be temporary as (IIRC) with `anyio` there will be no need for `aiofiles`, right?

Reply:
yep we can use anyio.Path and anyio.AsyncFile but we need to sort out the type annotations for anyio.AsyncFile before it can be used properly